src/app/public/modules/toast/types/toast-display-direction.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * @internal
```

Comment:
This is also exported by the `public_api`.

Reply:
This is only referenced from the above object ☝️ . Whatever we decided to do with `SkyToastContainerOptions` would impact this decision.